lib/Validator.php

```diff
@@ -12,6 +12,8 @@
     const XSD_FATTURA_SEMPLIFICATA_1_0      = 'Schema_VFSM10.xsd';
     const XSD_FATTURA_SEMPLIFICATA_LATEST   = 'Schema_VFSM10.xsd';
 
+    const XSD_MESSAGGI                      = 'MessaggiTypes_v1.1.xsd';
```

Comment:
We need to provide both specific and alias constants
```suggestion
    const XSD_MESSAGGI_1_0                  = 'MessaggiFatturaTypes_v1.0.xsd';
    const XSD_MESSAGGI_1_1                  = 'MessaggiTypes_v1.1.xsd';
    const XSD_MESSAGGI_LATEST               = 'MessaggiTypes_v1.1.xsd';
```